ucsb-cs156-w24/team03-w24-5pm-3
# Created Page for HelpRequest plus tests and stories

In this page, I created a page for HelpRequest and the corresponding tests and stories.
Steps for testing:
Log in
Navigate to the HelpReqest create page
Enter the desired data and press submit
Go to swagger-ui and check if data is entered correctly in the database

<img width="1195" alt="Screenshot 2024-02-21 at 4 53 19 AM" src="https://github.com/ucsb-cs156-w24/team03-w24-5pm-3/assets/97635056/e9d4271c-5b45-4152-b016-5b7b0103540e">

<img width="1197" alt="Screenshot 2024-02-21 at 4 54 45 AM" src="https://github.com/ucsb-cs156-w24/team03-w24-5pm-3/assets/97635056/0cd02f3f-15fd-4ffb-a9eb-010b556cfe87">


https://ucsb-cs156-w24.github.io/team03-w24-5pm-3/prs/105/

Closes #49.
